eq-author/src/components/ContentPicker/GroupContentPicker.js

```diff
@@ -20,6 +20,33 @@ const ActionButtons = styled(ButtonGroup)`
   flex: 0 0 auto;
 `;
 
+const generateDefaultState = (selectedObj, config) => {
+  if (!selectedObj) {
+    return {
+      openLevel: null,
+      selectedItem: null,
+    };
+  }
+  let openLevel;
+  let selectedItem;
+  forEach(selectedObj, value => {
+    if (value === null) {
+      return;
+    }
+    if (typeof value !== "object") {
+      openLevel = null;
+    } else {
+      openLevel = findIndex(config, ({ id }) => id === value.__typename);
```

Comment:
If `id === typename`? That looks weird - either its an id or a typename.

Reply:
Yeah the id of the config dictates what entity the config refers to, could add another field to the config with the same value? This whole group content picker feels very awkward here but this seemed to be the best way to set it up without making it specific to routing?

Reply:
Yeah, that might be a reasonable solution. Remove the idea of the group content picker and move it all into routing. But that can be looked at later.